arangod/Cluster/ClusterInfo.cpp

```diff
@@ -2441,15 +2469,13 @@ int ClusterInfo::ensureIndexCoordinator(
     do {
       resultBuilder.clear();
       errorCode = ensureIndexCoordinatorWithoutRollback(
-        databaseName, collectionID, idString, slice, create,
-        resultBuilder, errorMsg, timeout);
+        databaseName, collectionID, idString, slice, create, resultBuilder,
+        errorMsg, timeout);
       
-      if (errorCode == TRI_ERROR_NO_ERROR) {
-        return errorCode; // done
-      } else if (errorCode == TRI_ERROR_HTTP_PRECONDITION_FAILED) {
+      if (errorCode == TRI_ERROR_HTTP_PRECONDITION_FAILED) {
         auto diff = std::chrono::steady_clock::now() - start;
         if (diff < std::chrono::seconds(120)) {
-          uint32_t wt = RandomGenerator::interval(static_cast<uint32_t>(5000));
+          uint32_t wt = RandomGenerator::interval(static_cast<uint32_t>(1000));
```

Comment:
5 seconds better?

Reply:
seriously no point in waiting that long.